src/woo_publications/utils/tests/vcr.py

```diff
@@ -0,0 +1,29 @@
+import os
+from pathlib import Path
+
+from vcr.unittest import VCRMixin
+
+RECORD_MODE = os.environ.get("VCR_RECORD_MODE", "none")
+
+
+class VCRMixin(VCRMixin):
+    """
+    Mixin to use VCR in your unit tests.
+
+    Using this mixin will result in HTTP requests/responses being recorded.
+    """
+
+    def _get_cassette_library_dir(self):
+        split_path = str(self.__class__.__module__).split(".")
+        test_dir = split_path.pop()
+        path = Path("/".join(split_path))
```

Comment:
There's an easier way:

```py
import inspect
path = Path(inspect.getfile(self.__class__)).parent
```
which gives you the fully qualified path, so you can drop the `src` prefix even :)

```py
return str(path.parent / "vcr_casettes" / path.stem / class_name)
```

(untested)